src/vm/methodtable.cpp

```diff
@@ -9898,7 +9915,7 @@ MethodTable::TryResolveConstraintMethodApprox(
                 if (this->CanCastToInterface(pInterfaceMT))
                 {
                     // We can resolve to exact method
-                    pMD = this->GetMethodDescForInterfaceMethod(pInterfaceMT, pInterfaceMD);
+                    pMD = this->GetMethodDescForInterfaceMethod(pInterfaceMT, pInterfaceMD, FALSE /* throwOnConflict */);
                     _ASSERTE(pMD != NULL);
```

Comment:
Assertion no longer accurate

Reply:
fIsExactMethodResolved should be false if pMT is NULL

---
In reply to: [227113757](https://github.com/dotnet/coreclr/pull/20458#discussion_r227113757) [](ancestors = 227113757)

Reply:
Ah, yes. We would hit that assert now. Fixed.